packages/api-client/src/index.server.ts

```diff
@@ -37,6 +37,8 @@ import makeOrder from './api/makeOrder';
 import forgotPassword from './api/forgotPassword';
 import resetPassword from './api/resetPassword';
 
+const createAxiosFetcher = require('@spree/storefront-api-v2-sdk/dist/server/createAxiosFetcher').default;
```

Comment:
Shouldn't that be `import createAxiosFetcher from '@spree/storefront-api-v2-sdk/dist/server/createAxiosFetcher'`?